### PR TITLE
26-Create-Ruuter-based-SiGa-Service-Template

### PR DESCRIPTION
**Pull request by 1AhmedYasser**

### CODE REVIEW

File: DSL/Ruuter/POST/templates/siga.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+missing_type:
+  status: 400
+  return: "Sign type is missing"
+  next: end
+
+missing_parameters:
+  status: 400
+  return: "Missing Parameters"
+  next: end
```
Comment (trevorling) [inline DSL/Ruuter/POST/templates/siga.yml:84]: Please add a new line at the end on file.

File: docker-compose-siga.yml [hunk 2]
```diff
@@ -0,0 +1,166 @@
+version: "3.7"
+
+services:
+
```
Comment (trevorling) [inline docker-compose-siga.yml:4]: Please remove the an empty line (line 4)

File: docker-compose-siga.yml [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+    external: true
+    name: siga-logs-01
+  siga-logs-02:
+    external: true
+    name: siga-logs-02
+
+networks:
+  bykstack:
+    name: bykstack
+    driver: bridge
```
Comment (trevorling) [inline docker-compose-siga.yml:166]: Please add an empty line at the end of the file.

**Review by trevorling — CHANGES_REQUESTED** [review]

**Review by Minwasko — APPROVED** [review]

File: DSL/Ruuter/POST/templates/siga.yml (hunk 1 above)
Comment (1AhmedYasser) [reply DSL/Ruuter/POST/templates/siga.yml:84]: This one was already added

File: docker-compose-siga.yml (hunk 3 above)
Comment (1AhmedYasser) [reply docker-compose-siga.yml:165]: Added it

**Review by trevorling — APPROVED** [review]

**Comment by trevorling** [comment]
@1AhmedYasser 
Please resolve conflicts

**Review by trevorling — APPROVED** [review]

**Pull request merged**